hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/PrisonerSearchResource.kt

```diff
@@ -122,19 +127,11 @@ class PrisonerSearchResource(
     ) includeRestrictedPatients: Boolean,
     @RequestParam(value = "responseFields", required = false)
     @Parameter(
-      description = "*** BETA *** A list of fields to populate on the Prisoner record returned in the response. An empty list defaults to all fields. Note that this is currently being piloted in beta and should not be used yet. Please contact #syscon-devs if you wish to use this feature.",
+      description = "A list of fields to populate on the Prisoner record returned in the response. An empty list defaults to all fields.",
       example = "[prisonerNumber,firstName,aliases.firstName,currentIncentive.level.code]",
     )
     responseFields: List<String>? = null,
     @ParameterObject @PageableDefault
     pageable: Pageable,
-  ): Page<Prisoner> {
-    responseFields?.run {
-      responseFieldsValidator.findMissing(responseFields)
-        .takeIf { it.isNotEmpty() }
-        ?.run { throw BadRequestException("Invalid response fields requested: $this") }
-    }
```

Comment:
Pushed the throw into the validator and moved the validator into the service layer.